images/runtime/php-fpm/nginx_conf/default.conf

```diff
@@ -6,6 +6,7 @@ server {
     root /home/site/wwwroot;
     index  index.php index.html index.htm;
     server_name  example.com www.example.com; 
+    port_in_redirect off;
```

Comment:
Thanks for looking into it. Could you explain why we made this change, which issue it's trying to resolve?

Reply:
If this is not set, the nginx port is included in the `Location:` header sent back in 301/302 redirect responses.  This winds up going all the way back to the client browser, which then attempts to go to a non-existent port.

e.g.
```
$ curl -D - http://ericstensite1.ericsten2.antares-test.windows-int.net/banana
HTTP/1.1 301 Moved Permanently
Content-Length: 185
Content-Type: text/html
Date: Tue, 03 May 2022 19:06:47 GMT
Server: nginx/1.14.2
Location: http://ericstensite1.ericsten2.antares-test.windows-int.net:8080/banana/
```
The `:8080` shouldn't be returned.

Reply:
FYI: Service Desk case #2204080030000257

Reply:
Thanks @EricSten-MSFT for the case id. Triggering a nightly build for this PR right now to make sure nothing break for this change.